docs/windows.md

```diff
@@ -0,0 +1,32 @@
+# Windows Installation
+To setup and install PaSh on a Windows machine, `Windows Subsystem for Linux (WSL)` needs to be installed.
+
+## What is WSL
+The Windows Subsystem for Linux lets developers run a GNU/Linux environment 
+-- including most command-line tools, utilities, and applications -- 
+directly on Windows, unmodified, without the overhead of a traditional virtual 
+machine or dualboot setup.
+
+## Setup WSL
```

Comment:
I am not sure we should include such information in our documentation. The reason is that these instructions might quickly get outdated. Can we instead point to an official WSL installation link, e.g., https://docs.microsoft.com/en-us/windows/wsl/install-win10